Package.swift

```diff
@@ -124,7 +124,7 @@ if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
         .package(name: "swift-markdown", url: "https://github.com/apple/swift-markdown.git", .branch("main")),
         .package(name: "CLMDB", url: "https://github.com/apple/swift-lmdb.git", .branch("main")),
         .package(url: "https://github.com/apple/swift-argument-parser", .upToNextMinor(from: "1.0.1")),
-        .package(name: "SymbolKit", url: "https://github.com/apple/swift-docc-symbolkit", .branch("main")),
+        .package(name: "SymbolKit", url: "https://github.com/apple/swift-docc-symbolkit", .branch("revert-25-revert-23-5.7/objc-kinds")),
```

Comment:
When we update this to point at `release/5.7` can we also update `CLMDB` and `swift-markdown` to point at `release/5.7`? This shouldn't change the commit hash since `main` and `release/5.7` point to the same commit but we should still be pointing to the correct branch here.

Reply:
Yeah! I can do that once i land the SymbolKit PR.